spec/truffle/graal/aot_spec.rb

```diff
@@ -8,10 +8,10 @@
 
 require_relative '../../ruby/spec_helper'
 
-describe "Truffle::Graal.substrate?" do
+describe "Truffle::Graal.aot?" do
   
   it "returns a Boolean value" do
-    Truffle::Graal.substrate?.should be_true_or_false
+    Truffle::Graal.aot?.should be_true_or_false
```

Comment:
Since we are renaming this, I would propose to also move or alias them under Truffle directly instead of Truffle::Graal:
```ruby
Truffle.graal?
Truffle.aot?
```